conda_build/windows.py

```diff
@@ -121,7 +121,6 @@ def build(m):
             os.makedirs(path)
 
     src_dir = source.get_dir()
-    bld_bat = join(m.path, 'bld.bat')
```

Comment:
Is this code right?  You require bld_bat as an argument, but then you overwrite it here?